guides/Testing Community Of Practice

```diff
@@ -0,0 +1,17 @@
+---
+category: Testing Community Of Practice
+expires: 2020-01-04 
+---
+# Testing Guidelines
```

Comment:
I would use this as the title for you CoP and create another title for testing guidelines